src/components/LocationFilter/index.js

```diff
@@ -3,23 +3,28 @@ import styled from "styled-components";
 import { AppContext } from "../AppContext/AppContext";
 import IconSelected from "../../images/icon-selected.svg";
 import { BREAKPOINTS } from "../../constants";
+import { useHistory } from "react-router-dom";
 
 function LocationFilter() {
+  const history = useHistory();
   const {
     data,
     locations,
     setSelectedLocation,
     selectedLocation,
     setFilteredData,
+    setSelectedIndex,
   } = useContext(AppContext);
 
   const filterDataByLocation = (location) => {
     return data.filter((provider) => provider["provider town/city"] === location)
   }
 
   const handleLocationClick = (e, location) => {
+    history.push(`/`);
```

Comment:
if we set the selectedIndex to null, does change the route automatically? kind of feels like it should